pkgs/shells/nushell/plugins/hcl.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  IOKit,
+  CoreFoundation,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nu_plugin_hcl";
+  version = "0.100.0";
+
+  src = fetchFromGitHub {
+    owner = "yethal";
+    repo = pname;
+    rev = "2be6d719a0721981ad4a87931547324271fce5b5";
+    hash = "sha256-Uj4zzRJ1MtUZBCGc1/wCdh7ZOqLHPQhpdVJBseQR1uE=";
+  };
+
+  cargoHash = "sha256-J4i4lDoXcwU5ecyXiHh/4wwQGqp+dkQkwFHsCr2oNFw=";
+
+  nativeBuildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ rustPlatform.bindgenHook ];
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
```

Comment:
it's stylistic, but I think it would look better with newline between this lines